src/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.props

```
@@ -0,0 +1,5 @@
+<Project>
+  <PropertyGroup>
+    <PackageValidationDllPath>$(MSBuildThisFileDirectory)\..\tools\net5.0\Microsoft.DotNet.PackageValidation.dll</PackageValidationDllPath>
```

```suggestion
    <DotNetPackageValidationAssembly>$(MSBuildThisFileDirectory)\..\tools\net5.0\Microsoft.DotNet.PackageValidation.dll</DotNetPackageValidationAssembly>
```

to be consistent with other places like the TFM SKD: https://github.com/dotnet/arcade/blob/6d7ec926c8ae962ec5c1f5beeef3f67f9eeda635/src/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk/src/build/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk.props#L11-L12.